docs/source/guide/tutorials/base/how_to_train/anomaly_detection.rst

```diff
@@ -1,4 +1,190 @@
+    [INFO] 2023-02-21 16:42:43,640 - otx.algorithms.anomaly.tasks.inference - Exporting the OpenVINO model.
+    [ INFO ] The model was converted to IR v11, the latest model format that corresponds to the source DL framework input/output format. While IR v11 is backwards compatible with OpenVINO Inference Engine API v1.0, please use API v2.0 (as of 2022.1) to take advantage of the latest improvements in IR v11.
+    Find more information about API v2.0 and IR v11 at https://docs.openvino.ai/latest/openvino_2_0_transition_guide.html
+    [ SUCCESS ] Generated IR version 11 model.
+    [ SUCCESS ] XML file: /tmp/otx-anomaliba3imqkmo/onnx_model.xml
+    [ SUCCESS ] BIN file: /tmp/otx-anomaliba3imqkmo/onnx_model.bin
+
+Now that we have the exported model, let's check its performance using ``otx eval``
+
+.. code-block:: bash
+
+    otx eval ote_anomaly_detection_padim \
+        --test-data-roots data/anomaly/hazelnut_toy \
+        --load-weights otx-workspace-ANOMALY_DETECTION/openvino_models/openvino.xml \
+        --save-performance otx-workspace-ANOMALY_DETECTION/openvino_models/performance.json
+
+This gives the following results::
+
+    MultiScorePerformance(score: 0.8974358974358974, primary_metric: None, additional_metrics: (1 metrics), dashboard: (1 metric groups))
+
+************
+Optimization
+************
+
+Anomaly tasks can be optimized either in POT or NNCF format. For more information refer to the :doc:`optimization explanation <../../../explanation/additional_features/models_optimization>` section.
+
+
+----------------
+POT Optimization
+----------------
+
+Let's start with POT optimization.
+
+.. code-block:: bash
+
+    otx optimize ote_anomaly_detection_padim \
+        --train-data-roots data/anomaly/hazelnut_toy/ \
+        --load-weights otx-workspace-ANOMALY_DETECTION/openvino_models/openvino.xml \
+        --save-model-to otx-workspace-ANOMALY_DETECTION/pot_model
+
+-----------------
+NNCF Optimization
+-----------------
+
+To perform NNCF optimization, pass the torch ``pth`` weights to the ``opitmize`` command.
+
+.. code-block:: bash
+
+    otx optimize ote_anomaly_detection_padim \
+        --train-data-roots data/anomaly/hazelnut_toy/ \
+        --load-weights otx-workspace-ANOMALY_DETECTION/models/weights.pth \
+        --save-model-to otx-workspace-ANOMALY_DETECTION/nncf_model
+
+
+*******************************
+Segmentation and Classification
+*******************************
+
+While the above example shows Anomaly Detection, you can also train Anomaly Segmentation and Classification models. To see what tasks are available, you can pass ``anomaly_segmentation`` and ``anomaly_classification`` to ``otx find`` mentioned in the `Training`_ section. You can then use the same commands to train, evaluate, export and optimize the models.
```

Comment:
What about dataset? Dataset format and dataset itself is the same?

Reply:
I have updated to use the standard MVTec dataset